src/gtk/toga_gtk/font.py

```diff
@@ -1,5 +1,5 @@
-
 import gi
+
 gi.require_version("Pango", "1.0")
 from gi.repository import Pango
```

Comment:
Is Pango guaranteed to be installed if GTK is there? Do we need protections around this import like we do for Canvas and Webkit?